CMakeLists.txt

```diff
@@ -1360,8 +1360,8 @@ _add_if("brotli"        HAVE_BROTLI)
 _add_if("zstd"          HAVE_ZSTD)
 _add_if("AsynchDNS"     USE_ARES OR USE_THREADS_POSIX OR USE_THREADS_WIN32)
 _add_if("IDN"           HAVE_LIBIDN2)
-_add_if("Largefile"     (CURL_SIZEOF_CURL_OFF_T GREATER 4) AND
-                        ((SIZEOF_OFF_T GREATER 4) OR USE_WIN32_LARGE_FILES))
+_add_if("Largefile"     (SIZEOF_CURL_OFF_T GREATER "4") AND
+                        ((SIZEOF_OFF_T GREATER "4") OR USE_WIN32_LARGE_FILES))
```

Comment:
Hm, I don't think this is correct. The check worked without quotes in CMake, but `version.c` is somehow not detecting Largefile support. It seems like the detected `SIZEOF_CURL_OFF_T` may not be passed to `curl_config.h` correctly.

Reply:
oh right...